### PR TITLE
[Frontend] Tweak markup

### PR DESCRIPTION
**Pull request by charbelrami**
- replace tabs by 4 spaces
- move `lang` attribute from `head` to `html`
- `doctype` and `charset` are case insensitive
- `type="text/javascript"` is default

Thanks

### CODE REVIEW

**Comment by akhenry** [comment]
@charbelrami This is great, thanks very much. Before we can merge anything we just need a form signed by you. If you could send an email to ARC-DL-OpenMCT@mail.nasa.gov we will send you the form. Thanks again!

File: platform/forms/res/templates/toolbar.html [hunk 1]
```diff
@@ -20,31 +20,31 @@
...
+                             ng-required="item.required"
+                             ng-pattern="getRegExp(item.pattern)"
+                             options="item.options"
+                             structure="item"
+                             field="item.key">
+                </mct-control>
+            </ng-form>
+        </span>
+    </div>
 </form>
```
Comment (akhenry) [inline platform/forms/res/templates/toolbar.html]: Could you do me a favor and add a newline here? Thanks!

**Comment by akhenry** [comment]
@charbelrami Thanks for this. Changes look ok to me. Just one comment on a pre-existing missing newline at the end of `toolbar.html`.

Thanks again!

**Comment by charbelrami** [comment]
@akhenry I'm sorry, I've deleted the fork used for this PR, so I can't update it. Is OK to send another one adding the missing newline?
Thank you

**Comment by akhenry** [comment]
@charbelrami No problem at all. It's a pre-existing problem anyway, so I'll merge it as. Thanks for your help.

### Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? N/A
3. Code style and in-line documentation are appropriate? N/A
4. Commit messages meet standards? Y

**Comment by charbelrami** [comment]
Thank you!

**Pull request merged**